packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.jsx

```diff
@@ -120,6 +121,10 @@ const BuySellModal = () => {
     const history = useHistory();
     const location = useLocation();
 
+    const table_type = buy_sell_store.show_advertiser_page
```

Comment:
can we destructure these? :grin: 

const { show_advertiser_page, ... } = buy_sell_store
....

Reply:
explained at the bottom 🙏